File: life-program-institute/src/main/java/com/belphegor/lifeprograminstitute/controller/TestController.java
package com.belphegor.lifeprograminstitute.controller;

import com.belphegor.common.entity.ResponseEntity;
import com.belphegor.lifeprograminstitute.dao.UserDAO;
import com.belphegor.lifeprograminstitute.entity.User;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/test")
public class TestController {

    private static final Logger log = LoggerFactory.getLogger(TestController.class);

    @Autowired
    UserDAO userDAO;

    @RequestMapping("/hello")
    @ResponseBody
    public ResponseEntity hello(){
        log.info("testeeee1");
        return ResponseEntity.success("hello");
    }

}

File: src/main/kotlin/com/web/kotlinspring/controller/TestController.kt
package com.web.kotlinspring.controllerimport com.web.kotlinspring.reqparam.TestReqimport com.web.kotlinspring.reqparam.UserReqimport com.web.kotlinspring.service.UserServiceimport org.springframework.beans.factory.annotation.Autowiredimport org.springframework.http.MediaTypeimport org.springframework.web.bind.annotation.RequestMappingimport org.springframework.web.bind.annotation.RequestMethodimport org.springframework.web.bind.annotation.ResponseBodyimport org.springframework.web.bind.annotation.RestControllerimport src.main.kotlin.com.web.common.utils.ResponseDataimport src.main.kotlin.com.web.common.utils.success@RestController@RequestMapping("/life-blog")class TestController:BaseController(){    @Autowired    lateinit var userService:UserService    @RequestMapping(value = ["/test"],method = [RequestMethod.GET],produces = [MediaType.APPLICATION_JSON_UTF8_VALUE])    @ResponseBody    fun test(testReq: TestReq):ResponseData{        var str = "test Result:username:${testReq.username},password:${testReq.password}"        return success(str)    }    @RequestMapping(value = ["/modifyUserInfo"],method = [RequestMethod.GET],produces = [MediaType.APPLICATION_JSON_UTF8_VALUE])    @ResponseBody    fun modifyUserInfo(userReq: UserReq):ResponseData{        return userService.modifyUserInfo(userReq)    }}